implementations/rust/ockam/ockam_node/src/router/record.rs

```diff
@@ -283,8 +283,9 @@ impl AddressRecord {
         }
     }
 
+    #[inline]
     pub fn increment_msg_count(&self) {
-        self.msg_count.fetch_add(1, Ordering::Acquire);
+        self.msg_count.fetch_add(1, Ordering::Relaxed);
```

Comment:
Just curious about this one. What's the benefit of using Relaxed over Acquire here?

Reply:
When having multiple CPU cores, you may have multiple caches, and the one CPU may see variable A change but not B, and another one the reverse. A proper ordering guarantees the order of change when multiple variables are changed (outside of the atomics). But we aren't concerned about change order here.
The atomicity and consistency is always guaranteed regardless of the ordering. This only avoids an extra cache flush.